pkg/surveyext/editor.go

```diff
@@ -104,7 +104,9 @@ func (e *GhEditor) prompt(initialValue string, config *survey.PromptConfig) (int
 			break
 		}
 		if r == '\r' || r == '\n' {
-			if e.BlankAllowed {
+			if e.BlankAllowed && e.AppendDefault {
```

Comment:
It also might make sense to rename `BlankAllowed` to something like `SkipEditorAllowed`, since skipping no longer just returns an empty string.